superset/databases/schemas.py

```diff
@@ -70,6 +71,11 @@
     "all database schemas. For large data warehouse with thousands of "
     "tables, this can be expensive and put strain on the system."
 )  # pylint: disable=invalid-name
+configuration_method_description = (
+    "Configuration_method is used on the frontend to"
+    " inform the backend whether to explode parameters"
+    " or to provide only a sqlalchemy_uri."
```

Comment:
Just of consistency:

```suggestion
    "Configuration_method is used on the frontend to "
    "inform the backend whether to explode parameters "
    "or to provide only a sqlalchemy_uri."
```